enatega-multivendor-app/src/components/ItemCards/ItemCards.js

```diff
@@ -44,14 +44,17 @@ const ItemCard = ({ item, onPressItem, restaurant, tagCart }) => {
             <Text style={{ color: '#1C1C1E', fontSize: scale(12) }}>
               {`${configuration.currencySymbol}${item.variations[0].price}`}
             </Text>
-            <Text
+            {item.variations[0].discounted > 0 && (
```

Comment:
Variation can be empty. Apply nullable check item?.variations[0]?.discounted

Reply:
Thanks for the review, Optional chaining is added.